lib/actions/push.ts

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2016-2017 Resin.io
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { CommandDefinition } from 'capitano';
+import { ResinSDK } from 'resin-sdk';
+
+const PUSH_HELP_TEXT =
+`This command can be used to start a build on the remote resin.io cloud
+builders. The given source directory will be added to a tarfile, which
+will be sent to the resin.io builder, and the build will proceed. This
+can be used as a drop-in replacement for git push to deploy.`;
```

Comment:
We also normally include an example command in the help text. This is what becomes the CLI docs on https://docs.resin.io/reference/cli/, and they get nicely rendered there (and they're useful here too).

https://github.com/resin-io/resin-cli/blob/b7214a306c6b0a5e45387f93c9142219ff6d7b60/lib/actions/device.coffee#L133-L143 is a nice example imo.